pkg/traceability/muleemitter.go

```diff
@@ -63,18 +73,49 @@ func (me *MuleEventEmitter) Start() error {
 		return err
 	}
 
+	var lastTime time.Time
+	lastTime, err = time.Parse(time.RFC3339, strStartTime)
+	if err != nil {
+		logrus.WithError(err).Error("Unable to Parse Last Time")
+		return err
```

Comment:
If the last time can't be parsed, then we aren't publishing any events. Is that what we want?

Reply:
No, what do you suggest?